src/game_api/entities_items.hpp

```diff
@@ -739,7 +765,8 @@ class RollingItem : public Purchasable
   public:
     float roll_speed; // only positive numbers
 
-    virtual void on_purchase(Entity* who, int32_t unknown) = 0; // give you the powerup if you buy it
+    /// Skip this function for item to be unpickable
+    virtual void give_powerup(Entity* who, bool) = 0;
```

Comment:
Well uh, now I found an issue here, skipping it doesn't seem to make pickups unpickable, and the boolean parameter seems to be for playing the sound or not (calling it with true plays the sound, with false it doesn't play). Probably named something like `play_sound` or `play_sfx`

Reply:
Skipping it does prevent anything from happening when using `E:equip(P)` though

Reply:
fixed

Reply:
no name given for the bool parameter?

Reply:
i forgor